src/test/resources/com/jpinpoint/perf/lang/java/ruleset/common/xml/AvoidUnconditionalBuiltLogStrings.xml

```diff
@@ -308,6 +308,26 @@ public class MyProducer {
     }
 
 
+}
+]]></code>
+    </test-code>
+
+    <test-code>
+        <description>false positive in AvoidUnconditionalBuiltLogStrings #430</description>
+        <expected-problems>0</expected-problems>
+        <code><![CDATA[
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+class Issue430 {
+    String bar() {
+        StringBuilder messageBuilder = new StringBuilder();
+        messageBuilder.append("-"); // <-- false positive
+
+        log.info("message: {}" , messageBuilder);
```

Comment:
here in this particular code case: when info level is active, the messageBuilder.toString() is actually called twice: for logging and the return value... might be for another rule, there is some unnecessary overhead. But might be difficult to catch and not worth the effort for a rule  (?)

Reply:
yeah, good point, maybe worth a rule, but not high priority